invite_service/repository/mock_invite_repository.py

```diff
@@ -161,6 +183,7 @@ async def create_invite(self, invite: Invite) -> None:
 
         """
         invite.id = str(uuid4())
+        invite.created_at = datetime.now()
```

Comment:
add "invite.deleted_at = None" at the next line

Reply:
fixed